src/CopyrighFixer/CopyrighFixer/worker.h

```diff
@@ -32,6 +38,10 @@ class CopyrighFixer_EXPORT Worker {
      */
     void printHelp() const;
 
+protected:
+    Config *conf_;
+    ConfigParser *confParser_;
+    Signer *subscriber_;
```

Comment:
you forget to add default values of the pointers.
```suggestion
    Config *conf_ = nullptr;
    ConfigParser *confParser_ = nullptr;
    Signer *subscriber_ = nullptr;
```